lighthouse-cli/test/smokehouse/report-assert.js

```diff
@@ -225,23 +225,37 @@ function pruneExpectations(localConsole, lhr, expected, reportOptions) {
   const isFraggleRock = reportOptions?.useFraggleRock;
   const isBundled = reportOptions?.isBundled;
 
+  /**
```

Comment:
is it possible to tighten this up? It's a version number check but spread over like four functions :)